pkg/postgres/error.go

```diff
@@ -11,6 +11,8 @@ func toErrox(err error) error {
 		switch pgErr.Code {
```

Comment:
It will be useful to link https://www.postgresql.org/docs/current/errcodes-appendix.html in comment to this function so that future-us can maintain this file more easily.